mlflow/environment_variables.py

```diff
@@ -977,3 +977,27 @@ def _split_strip(s: str) -> list[str]:
 MLFLOW_ENFORCE_STDIN_SCORING_SERVER_FOR_SPARK_UDF = _BooleanEnvironmentVariable(
     "MLFLOW_ENFORCE_STDIN_SCORING_SERVER_FOR_SPARK_UDF", True
 )
+
+#: Sepecifies whether to enable job execution feature for MLflow server.
+#: This feature requires "huey" package dependency, and requires Mlflow server configures
+#: --backend-store-uri to database URI.
+#: (default: ``False``)
+MLFLOW_SERVER_ENABLE_JOB_EXECUTION = _BooleanEnvironmentVariable(
+    "MLFLOW_SERVER_ENABLE_JOB_EXECUTION", False
+)
```

Comment:
Can we default to `True` so that users can start using job-based functionality (e.g. online scorers) by default?

Reply:
rethinking on this , the file-store backend is already deprecated and will be removed soon. I think we don't need to waste time to implement file-store backend for Job operations